tests/phpunit/tests/rest-api/rest-server.php

```diff
@@ -6,6 +6,8 @@
  * @subpackage REST API
  *
  * @group restapi
+ *
+ * @coversDefaultClass WP_REST_Server
```

Comment:
With the planned restructuring of the test suite, adding new `@coversDefaultClass` annotations will make it more difficult to reorganise individual test methods into the new structure.

Can we remove this annotation and use `@covers WP_REST_Server::get_index` directly in the test method docblock?